src/any_agent/tracing/instrumentation/agno.py

```diff
@@ -0,0 +1,124 @@
+# mypy: disable-error-code="no-untyped-call, no-untyped-def"
+from __future__ import annotations
+
+import json
+from typing import TYPE_CHECKING
+
+from opentelemetry.trace import StatusCode
+from wrapt.patches import resolve_path, wrap_function_wrapper
+
+if TYPE_CHECKING:
+    from opentelemetry.trace import Span, Tracer
+
+
+class _AgnoInstrumentor:
+    def instrument(self, tracer: Tracer) -> None:
+        async def wrap_aprocess_model_response(
+            wrapped,
+            instance,
+            args,
+            kwargs,
+        ):
+            with tracer.start_as_current_span(f"call_llm {instance.id}") as span:
+                span.set_attributes(
+                    {
+                        "gen_ai.operation.name": "call_llm",
+                        "gen_ai.request.model": instance.id,
+                    }
+                )
+
+                assistant_message, has_tool_calls = await wrapped(*args, **kwargs)
+
+                if assistant_message.content:
+                    span.set_attributes(
```

Comment:
idk can there be some check here to add "genai.input" with the prompt if it's the first call_llm span?